integration_tests/steps/user_profile.py

```diff
@@ -0,0 +1,122 @@
+import uuid
+
+import d3a.constants
+from behave import given, then
+from d3a.d3a_core.util import d3a_path
+from d3a.models.config import SimulationConfig
+from d3a_interface.constants_limits import GlobalConfig
+from d3a_interface.read_user_profile import read_arbitrary_profile, InputProfileTypes
+from pendulum import duration
+
+
+@given("uuids are initiated")
+def init_uuids(context):
+    context.config_uuid = str(uuid.uuid4())
+    context.pv_area_uuid = str(uuid.uuid4())
+    context.pv_profile_uuid = str(uuid.uuid4())
+    context.load_area_uuid = str(uuid.uuid4())
+    context.load_profile_uuid = str(uuid.uuid4())
+    GlobalConfig.slot_length = duration(minutes=60)
+
+
+@given("a yearly PV profile exist in the DB")
```

Comment:
```suggestion
@given("an yearly PV profile exist in the DB")
```
nitpicky

Reply:
I think the grammatically correct version is the original one:
> "a yearly ..."

That's because "Y" has a consonant sound in this word (see [Which one is grammatically correct: "an year" or "a year"?](https://www.quora.com/Which-one-is-grammatically-correct-an-year-or-a-year#:~:text=Originally%20Answered%3A%20What%20is%20grammatically,%22a%20year%22%20is%20correct.))

Reply:
Wow, learned some english grammar today, was so used to of `an yearly`